Star Trek Adventurers 2d20/sheet.json

```diff
@@ -0,0 +1,7 @@
+{
+	"html": "Star Trek Adventures 2d20.html",
+	"css": "Star Trek Adventures 2d20.css",
+	"authors": "David Collier",
+	"preview": "Star Trek Adventures 2d20.PNG",
```

Comment:
I just went off the other json's  this is my first time adding to the official repository, most of my custom sheets are just for my games - where would you put your roll20id, authors?